docker/generic/start_proxy.py

```diff
@@ -182,6 +182,13 @@ def make_argparser():
         Cipher suites to use for downstream connections as a comma-separated list.
         Please refer to https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/auth/common.proto#auth-tlsparameters''')
 
+    parser.add_argument('--ssl_server_root_cert_path', default=None, help='''
+         The file path of root certificates that ESPv2 uses to do downstream client certificate-based authentication.
+        If not specified, ESPv2 doesn't it by default. 
+        
+        Before using this feature, please verify if your compute environment is compatible with it.
```

Comment:
what computer environment?   how to verify?, verify what?

Reply:
Just reworded. I wanna warn users that many GCP platforms will terminate TSL/mTSL in ingress.